src/pybind/mgr/dashboard/frontend/src/app/shared/components/refresh-selector/refresh-selector.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+
+import { RefreshIntervalService } from '../../services/refresh-interval.service';
+
+@Component({
+  selector: 'cd-refresh-selector',
+  templateUrl: './refresh-selector.component.html',
+  styleUrls: ['./refresh-selector.component.scss']
+})
+export class RefreshSelectorComponent implements OnInit {
+  selectedInterval: string;
+  intervalList: Array<string> = ['5s', '10s', '15s', '30s', '1min', '3min', '5min'];
```

Comment:
Please add a space between value and unit.

``The value of a quantity is written as a number followed by a space (representing a multiplication sign) and a unit symbol``, see https://en.wikipedia.org/wiki/International_System_of_Units.

Reply:
Good spotting. My apologies, I did not specify this correctly in the corresponding tracker issue.